operators/gitops-primer-operator/0.0.1/manifests/gitops-primer.clusterserviceversion.yaml

```
@@ -0,0 +1,396 @@
+                    port: 8081
+                  initialDelaySeconds: 15
+                  periodSeconds: 20
+                name: manager
+                readinessProbe:
+                  httpGet:
+                    path: /readyz
+                    port: 8081
+                  initialDelaySeconds: 5
+                  periodSeconds: 10
+                resources:
+                  limits:
+                    cpu: 100m
+                    memory: 300Mi
+                  requests:
+                    cpu: 100m
+                    memory: 20Mi
+                securityContext:
+                  allowPrivilegeEscalation: false
+              securityContext:
+                runAsNonRoot: true
+              serviceAccountName: gitops-primer-controller-manager
+              terminationGracePeriodSeconds: 10
+      - name: gitops-primer-mutating-webhook-deployment
+        spec:
+          replicas: 1
+          selector:
+            matchLabels:
+              app: export-webhook
+          strategy: {}
+          template:
+            metadata:
+              labels:
+                app: export-webhook
+            spec:
+              containers:
+              - image: quay.io/konveyor/gitops-primer-webhook:v0.0.1
+                imagePullPolicy: IfNotPresent
+                name: export-webhook
+                ports:
+                - containerPort: 8000
+                resources:
+                  limits:
+                    cpu: 500m
+                    memory: 128Mi
+                volumeMounts:
+                - mountPath: /tls
+                  name: export-tls-secret
+                  readOnly: true
+              volumes:
+              - name: export-tls-secret
+                secret:
+                  secretName: gitops-primer-mutating-webhook-deployment-service-cert
+      permissions:
+      - rules:
+        - apiGroups:
+          - ""
+          resources:
+          - configmaps
+          verbs:
+          - get
+          - list
+          - watch
+          - create
+          - update
+          - patch
+          - delete
+        - apiGroups:
+          - coordination.k8s.io
+          resources:
+          - leases
+          verbs:
+          - get
+          - list
+          - watch
+          - create
+          - update
+          - patch
+          - delete
+        - apiGroups:
+          - ""
+          resources:
+          - events
+          verbs:
+          - create
+          - patch
+        serviceAccountName: gitops-primer-controller-manager
+    strategy: deployment
+  installModes:
+  - supported: false
+    type: OwnNamespace
+  - supported: false
+    type: SingleNamespace
+  - supported: false
+    type: MultiNamespace
+  - supported: true
+    type: AllNamespaces
+  keywords:
+  - gitops
+  - operator
+  links:
+  - name: Gitops Primer
+    url: https://github.com/konveyor/gitops-primer
+  maturity: alpha
+  provider:
+    name: OCTO-ET
```

@jmontleon , Is this [still] correct for the provider name or should it [now] be `Konveyor` (or `Red Hat`)?

```suggestion
  provider:
    name: Konveyor
```

---

Looks like @cooktheryan has proposed `Konveyor` in a subsequent PR in [konveyor/gitops-primer#R360-R361](https://github.com/konveyor/gitops-primer/pull/157/files#diff-36d1afe19093400e25a5bbab91aab02bdff0a385377b7a48e629ad4208fcba11R360-R361)